src/components/shared/ButtonWithPanel.js

```diff
@@ -4,44 +4,37 @@
 
 // @flow
 
+// This implements a Button that trrigers a panel. The panel will have a small
```

Comment:
I'm fixing this typo in a later commit ;-)

Reply:
trrrrrigers